src/Stepper/Step.tsx

```diff
@@ -0,0 +1,111 @@
+import * as classnames from "classnames";
+import * as React from "react";
+import * as PropTypes from "prop-types";
+
+import "./Step.less";
+import CheckMark from "./CheckMark";
+import {FlexBox} from "../flex";
+import Exclamation from "./Exclamation";
+
+const propTypes = {
+  className: PropTypes.string,
+  title: PropTypes.string,
+  description: PropTypes.oneOfType([PropTypes.node]),
+  showTitle: PropTypes.bool,
+  showMessage: PropTypes.bool,
+  showDescription: PropTypes.bool,
+  id: PropTypes.number,
+  seekable: PropTypes.bool,
+  success: PropTypes.bool,
+  current: PropTypes.bool,
+  warning: PropTypes.bool,
```

Comment:
should `warning: true` and `success: true` be mutually exclusive?
i'm a little confused by the API on first glance, which is a sign it could be tightened up a bit.

from reading the props, i *think* `warning` enables warning styling (colour change, icon change, etc) and `success` does similar for a successful state. if that's the case, we can go with the pattern we've been using elsewhere and provide an enum of state values - `INCOMPLETE`, `SUCCESS`, `WARNING` - and require that the client specify one of those states

related: we're using this to decide whether or not the `message` is rendered as warning text - that's conflating 2 concepts. i'd recommend adding a `warning: PropTypes.string` instead along with `message: PropTypes.string` and displaying either/both depending on which is specified (unless `message` is always a warning, in which case, might as well leave out `message`)